src/lib/journey/callbacks/string-attribute/string-attribute-input.svelte

```diff
@@ -21,7 +21,7 @@
   } from '$journey/journey.interfaces';
   import type { styleSchema } from '$lib/style.store';
   import type { Maybe } from '$lib/interfaces';
-  import type { StringDict } from '@forgerock/javascript-sdk/src/shared/interfaces';
+  import type { StringDict } from '@forgerock/javascript-sdk/src/shared/interfaces.d.ts';
```

Comment:
I'm assuming this `.d.ts` is necessary, but just wanted to make sure. This seems to be unique relative to the other type imports.

Reply:
its not i will remove it, it was leftover i think from my initial issues with signals